content/guides/code-splitting-require.md

```diff
@@ -75,16 +83,99 @@ module.exports = function(env) {
         entry: './js/entry.js',
         output: {
             filename: 'bundle.js',
-            path: path.resolve(__dirname, 'dist')
+            path: path.resolve(__dirname, 'dist'),
+            publicPath: 'https://cdn.example.com/assets/',
+            // tell webpack where to load the on-demand bundles. 
+
+            pathinfo: true,
+            // show comments in bundles, just to beautify the output of this example.
+            // should not be used for production.
         }
     }
 }
+
 ```
+
+T> `output.publicPath` is an important option when using code-splitting, it is used to tell webpack where to load your bundles on-demand, see the [configuration documentation](/configuration/output/#output-publicpath).
+
 On running webpack on this project, we find that webpack has created two new bundles, `bundle.js` and `0.bundle.js`.
 
 `entry.js` and `a.js` are bundled in `bundle.js`.
 
-`b.js` is bundled in `0.bundle.js`.
+**bundle.js**
+
+```javascript
+//webpack bootstrap code...
```

Comment:
Since you specify the `publicPath`, please show the relevant bootstrap snippet which is affected by this. Similar to:
```js
// webpack bootstrap code...
/******/  // __webpack_public_path__
/******/  __webpack_require__.p = "https://cdn.example.com/assets/";
// webpack bootstrap code...
```